cmd/helm/status.go

```diff
@@ -144,7 +144,7 @@ func (s statusPrinter) WriteTable(out io.Writer) error {
 		_, _ = fmt.Fprintf(out, "DESCRIPTION: %s\n", s.release.Info.Description)
 	}
 
-	if s.showResources && s.release.Info.Resources != nil && len(s.release.Info.Resources) > 0 {
```

Comment:
this seems spurious / logically incorect to remove `s.showResources` here?

Reply:
Ah crap, I had accidentally started my branch from #13456 that removed the `--show-resources` flag, I rebased this PR earlier today but this line must have been changed in the wrong commit. I'll revert it now